mtest/libmscore/clef/updateReference

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
-
 cp ../../../build.debug/mtest/libmscore/clef/clef-1.mscx clef-1-ref.mscx
 cp ../../../build.debug/mtest/libmscore/clef/clef-2.mscx clef-2-ref.mscx
+cp ../../../build.debug/mtest/libmscore/clef/clef-3.mscx clef-3-ref.mscx
```

Comment:
It really helped me to update these updateReference scripts, just to get mtest working.  So, as a convenience, I've included them in the PR, along with the -ref.mscx file changes.  There are only a few of them.